src/hazelcore/CursorInfo_Exp.re

```diff
@@ -289,45 +282,26 @@ and syn_cursor_info_line =
     | None => None
     | Some(ci) => Some(CursorNotOnDeferredVarPat(ci))
     }
-  | LetLineZP(zp, ann, def) =>
-    let pat_ci = CursorInfo_Pat.ana_cursor_info(~steps=steps @ [0], ctx, zp);
-    switch (ann) {
-    | None =>
-      switch (Statics_Exp.syn(ctx, def)) {
-      | None => None
-      | Some(ty1) => pat_ci(ty1)
+  | LetLineZP(zp, def) =>
```

Comment:
this isn't making use of the synthesized type of the definition -- useful information for the user I think